### ISSUE TITLE
Request Instructions

### ISSUE THREAD

**Issue by liruiluo** [mokeddembillel/deep-symbolic-optimizer#1]
Thanks for a wonderful implementation!
I've been looking for a pytorch version of dso, could you please write a usage note for this repo?

**Comment by KrisJe** [comment]
Hi I tried your implementation. Great work! 

Question: How do I read this?

loss:  77.89010665994053
Expression:  ['+', '+', 'sin', 'x_0', 'c', '+', '*', 'x_0', 'c', 'c']
Constants:  [-4.00254724e-01  4.46392962e-09  3.40025462e+00]  

Also the higher the loss value the better the fit, but shouldn't loss go to zero in this case?

Thanks in advance.

**Comment by mokeddembillel** [comment]
Hi, Thanks. 

Actually as far as I remember, this repo is still not working properly. I implemented it purely based on the paper and I remember there were many tricks that went into the original code which is in tensorflow 1. The goal for me at that point was to understand the tensorflow code and see the differences between mine and their's so i can fix mine. This repo still needs some work, If you are interested in contributing, let me know. Thank you.
